src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java

```diff
@@ -43,6 +44,14 @@ public int testMethod4() {
         return 1;
     }
 
+    @GraphQLDeprecate("test field deprecated")
```

Comment:
make codecov happy by lifting those fields to the top of the file above the resolver functions